flink-java/pom.xml

```diff
@@ -64,6 +64,18 @@ under the License.
 			<version>0.5.1</version>
 		</dependency>
 
+		<dependency>
```

Comment:
They are actually optional dependencies. They are not included unless we explicitly include them.